integration_tests/mockApis/allocationManager.ts

```diff
@@ -21,7 +21,7 @@ export default {
     stubFor({
       request: {
         method: 'GET',
-        urlPattern: '/api/allocation/\\w+',
+        urlPattern: '/allocation-manager/api/allocation/\\w+',
```

Comment:
the path was wrong here, which is why Engelbert Humperdinck now needed adding anywhere POM is displayed - it should always have displayed, but the mock was not properly setup.